openrpc.json

```diff
@@ -64,9 +64,7 @@
               "value": {
                 "chainId": "0x64",
                 "chainName": "Gnosis",
-                "rpcUrls": [
-                  "https://dai.poa.network"
-                ],
+                "rpcUrls": ["https://dai.poa.network"],
```

Comment:
sorry for these random formatting changes... I can revert them if that's desired